components/n-ui/header/partials/header/sticky.html

```diff
@@ -43,6 +43,7 @@
 											class="o-header__nav-{{#ifEquals label 'Subscribe'}}button{{else}}link{{/ifEquals}}"
```

Comment:
Could we split this into two <li>'s now? There are now three places we check `{{#ifEquals label 'Subscribe'}}`